bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -249,6 +276,16 @@ limitations under the License.
           <transformers>
             <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
           </transformers>
+	  <filters>
+	      <filter>
+		  <artifact>*:*</artifact>
+		  <excludes>
+		      <exclude>META-INF/*.SF</exclude>
```

Comment:
What does this do?

Reply:
It is for avoiding errors like "User class threw exception: java.lang.SecurityException: Invalid signature file digest for Manifest main attributes" when creating a shaded jar file.

references: 
https://stackoverflow.com/questions/34855649/invalid-signature-file-digest-for-manifest-main-attributes-exception-while-tryin
https://maven.apache.org/plugins/maven-shade-plugin/examples/includes-excludes.html